flow/platforms/asap7/config.mk

```diff
@@ -267,3 +267,5 @@ export IR_DROP_LAYER ?= M1
 
 # Allow empty GDS cell
 export GDS_ALLOW_EMPTY ?= fakeram.*
+
+export PLACE_PINS_ARGS = -annealing
```

Comment:
Do you want every asap7 design running with annealing? We have some metrics failures in other asap7 designs that are probably caused by running them with annealing.

Perhaps we should change only the mock-array design?

Reply:
ah, this is a gaffe. i only intended to do this w mock array.